cloudbuild/e2e-test.yaml

```
@@ -44,7 +44,8 @@ steps:
       - VERTEX_NETWORK=${_TEST_VERTEX_NETWORK}
       - PIPELINE_TEMPLATE=${_PIPELINE_TEMPLATE}
       - VERTEX_PIPELINE_ROOT=${_TEST_VERTEX_PIPELINE_ROOT}
-      - PIPELINE_FILES_GCS_PATH=${_PIPELINE_PUBLISH_GCS_PATH}/${COMMIT_SHA}
+      - PIPELINE_FILES_GCS_PATH=${_PIPELINE_PUBLISH_GCS_PATH}/${COMMIT_SHA}/assets
+      - RESOURCE_SUFFIX=_${SHORT_SHA}
```

Would have expected:

```suggestion
      - RESOURCE_SUFFIX=_${COMMIT_SHA}
      - PIPELINE_FILES_GCS_PATH=${_PIPELINE_PUBLISH_GCS_PATH}/assets_${COMMIT_SHA}
```

---

The first build step that copies the assets, copies them into `${_PIPELINE_PUBLISH_GCS_PATH}/${COMMIT_SHA}`.

Since now `_PIPELINE_PUBLISH_GCS_PATH` is expected to have the `assets` folder in the code, I've added the extra directory in its definition